### DISCUSSION TITLE
list the vsphere resources which are created during IPI installation via govc

### DISCUSSION THREAD

**Discussion by vishnoisuresh** [okd-project/okd#638] [Ideas]
It would be convenient or improve the user experience If all of the resources which are created during IPI installation are fetchable via govc. When there is broken okd cluster sometimes it's easier to completely remove the cluster  and recreate it.

**Comment by elmiko** [comment]
related discussion from slack, https://kubernetes.slack.com/archives/C6BRQSH2S/p1620936073092900

**Comment by evertmulder** [comment]
Something like this? Where `okd4-7brg6` is the InfraID made from the cluster name +  installer generated part. 
```
govc tags.attached.ls okd4-7brg6 | xargs govc ls -L
```
